pcov_arginfo.h

```diff
@@ -0,0 +1,41 @@
+/* This is a generated file, edit the .stub.php file instead.
+ * Stub hash: d2cebacd7516b58ebb9ce549595ee2b8ce3c3d6d */
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_pcov_start, 0, 0, IS_VOID, 0)
+ZEND_END_ARG_INFO()
+
+#define arginfo_pcov_stop arginfo_pcov_start
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_pcov_collect, 0, 0, IS_ARRAY, 1)
+	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, type, IS_LONG, 0, "pcov\\all")
+	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, filter, IS_ARRAY, 0, "[]")
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_pcov_clear, 0, 0, IS_VOID, 0)
+	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, files, _IS_BOOL, 0, "false")
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_pcov_waiting, 0, 0, IS_ARRAY, 1)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_pcov_memory, 0, 0, IS_LONG, 1)
+ZEND_END_ARG_INFO()
+
+
+ZEND_FUNCTION(start);
+ZEND_FUNCTION(stop);
+ZEND_FUNCTION(collect);
+ZEND_FUNCTION(clear);
+ZEND_FUNCTION(waiting);
+ZEND_FUNCTION(memory);
+
+
+static const zend_function_entry ext_functions[] = {
```

Comment:
why `ext_functions` and not something with `pcov` in the name ?

Reply:
> why `ext_functions` and not something with `pcov` in the name ?

This file is generated ;)
So, a PHP choice, not under control of this extension